### PR TITLE
Add RAI

### PR DESCRIPTION
**Pull request by stefanionescu**
**Description**
Add the RAI stablecoin addresses for Optimism.

**Additional context**
RAI was deployed on Kovan & Mainnet Optimism and we now want to add its addresses here.

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -395,6 +395,50 @@
...
+      "symbol": "RAI",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/RAI.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0x76b06a2f6dF6f0514e7BEC52a9AfB3f603b477CD",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:423]: We need to be able to mint test tokens here using the `0xe3a90926133fef3c58514fA1aAeF75E0998aedD5` account, see [readme](https://github.com/ethereum-optimism/ethereum-optimism.github.io#contributing-guide) in repo

**Review by elenadimitrova — COMMENTED** [review]
Changes look good just one comment.

File: optimism.tokenlist.json (hunk 1 above)
Comment (stefanionescu) [reply optimism.tokenlist.json:423]: @elenadimitrova do you want to mint on Kovan or Kovan optimism? Anyone can mint RAI on Kovan using the app

https://app-kovan.reflexer.finance/

Comment (elenadimitrova) [reply optimism.tokenlist.json:423]: Thanks, managed to mint on Kovan for testing 👍

**Review by elenadimitrova — APPROVED** [review]

**Review by elenadimitrova — APPROVED** [review]

**Comment by elenadimitrova** [comment]
Thanks for the submission. Successfully deposited kovan RAI to L2 https://kovan-optimistic.etherscan.io/tx/0xb7d4317e509bde0fdd9309284de34304a83dfbbef28a2fcaae627bc38aeb820a
Would be nice to update the logo though as it's not displaying well.

**Pull request merged**